### PR TITLE
extend encounter with the badger to three paths

### PR DESCRIPTION
**Pull request by saacuna**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/coffee/nude-run/cold-water/cold-water.md [hunk 1]
```diff
@@ -4,3 +4,25 @@ head.
...
+You suddenly notice the .38 magnum lying on the ground in front of you. 
+It looks like there is one bullet left. locked and loaded. 
+
+Unfortunately the badger noticed the gun as well. The gun is closer 
+to you, but not by much. You might be able to grab it, But you are not
+sure how quick the badgers reflexes are.
+
+What do you do?
+
+["Strangle the badger with your hands"](../nude-run/cold-water/strangle/strangle.md)
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/cold-water.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/coffee/nude-run/cold-water/cold-water.md [hunk 2]
```diff
@@ -4,3 +4,25 @@ head.
...
+
+["Make a move for the gun"](../nude-run/cold-water/grab-gun/grab-gun.md)
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/cold-water.md]: This link is also invalid.

File: english/coffee/nude-run/cold-water/cold-water.md [hunk 3]
```diff
@@ -4,3 +4,25 @@ head.
...
+
+["Run away"](../nude-run/cold-water/run-away/run-away.md)
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/cold-water.md]: This link is also invalid.

File: english/coffee/nude-run/cold-water/grab-gun/grab-gun.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+You lunge for the gun.
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/grab-gun/grab-gun.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/coffee/nude-run/cold-water/run-away/run-away.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+You decide to run for it.
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/run-away/run-away.md]: This new file also has no link to it.

File: english/coffee/nude-run/cold-water/strangle/strangle.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+You decide to forget the gun and go for the badger with
```
Comment (casey-collab) [inline english/coffee/nude-run/cold-water/strangle/strangle.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/coffee/nude-run/cold-water/cold-water.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/coffee/nude-run/cold-water/cold-water.md]: I believe that using simply `strangle/strangle.md` as the link would work here. Similarly for the others.

**Comment by saacuna** [comment]
Thanks for the feedback! I'm so excited you actually interacted with me and this wasn't just a one-off video. I found the tutorial so helpful and am using Git in my research now! Why did I wait so long to do this?

**Comment by cbuckey-uda** [comment]
You're welcome. Glad you enjoyed the course!

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**